homeworks/rostyslav.havuka_RostyslavHavuka/5-rxjs/main.js

```diff
@@ -0,0 +1,20 @@
+const { fromEvent, animationFrameScheduler } = window.rxjs;
+const {
+    pairwise, map, distinctUntilChanged, throttleTime,
+} = window.rxjs.operators;
+
+const scroll = fromEvent(window, 'scroll');
+const header = document.querySelector('header');
+
+const watchScroll = scroll.pipe(
+    throttleTime(150, animationFrameScheduler),
+    map(() => window.pageYOffset),
+    pairwise(),
+    map(([y1, y2]) => (y2 < y1 ? 'Up' : 'Down')),
+    distinctUntilChanged(),
+);
+
+watchScroll.subscribe((event) => {
+    if (event === 'Up') header.classList.add('header-logo');
```

Comment:
It's better to avoid the `if` shortcut. It would look better if you change code like this:
```js
if (event === 'Up') {
  header.classList.add('header-logo');
}
```
The comment is applicable to the next if as well.

Also, you can use `switch-case` here